pkg/controller/accountclaim/reuse.go

```diff
@@ -96,6 +100,33 @@ func (r *ReconcileAccountClaim) resetAccountSpecStatus(reqLogger logr.Logger, re
 	return nil
 }
 
+func (r *ReconcileAccountClaim) recreateAccountSecrets(reqLogger logr.Logger, reusedAccount *awsv1alpha1.Account) error {
```

Comment:
Am I missing something? Does this method remove existing secrets and recreates them or simply removes existing secrets? Also is it used anywhere?

Reply:
Left over from some testing. Removing

Reply:
Will this be used?

Reply:
Forgot to remove it